Xwt.Gtk/Xwt.GtkBackend/WidgetBackend.cs

```diff
@@ -185,38 +185,23 @@ public void SetCursor (CursorType cursor)
 			CurrentCursor = cursor;
 			Gdk.Cursor gc;
 			if (!gtkCursors.TryGetValue (cursor, out gc)) {
-				Gdk.CursorType ctype;
-				if (cursor == CursorType.Arrow)
-					ctype = Gdk.CursorType.LeftPtr;
-				else if (cursor == CursorType.Crosshair)
-					ctype = Gdk.CursorType.Crosshair;
-				else if (cursor == CursorType.Hand)
-					ctype = Gdk.CursorType.Hand1;
-				else if (cursor == CursorType.IBeam)
-					ctype = Gdk.CursorType.Xterm;
-				else if (cursor == CursorType.ResizeDown)
-					ctype = Gdk.CursorType.BottomSide;
-				else if (cursor == CursorType.ResizeUp)
-					ctype = Gdk.CursorType.TopSide;
-				else if (cursor == CursorType.ResizeLeft)
-					ctype = Gdk.CursorType.LeftSide;
-				else if (cursor == CursorType.ResizeRight)
-					ctype = Gdk.CursorType.RightSide;
-				else if (cursor == CursorType.ResizeLeftRight)
-					ctype = Gdk.CursorType.SbHDoubleArrow;
-				else if (cursor == CursorType.ResizeUpDown)
-					ctype = Gdk.CursorType.SbVDoubleArrow;
-				else if (cursor == CursorType.Move)
-					ctype = Gdk.CursorType.Fleur;
-				else if (cursor == CursorType.Wait)
-					ctype = Gdk.CursorType.Watch;
-				else if (cursor == CursorType.Help)
-					ctype = Gdk.CursorType.QuestionArrow;
-				else if (cursor == CursorType.Invisible)
-					ctype = (Gdk.CursorType)(-2); // Gdk.CursorType.None, since Gtk 2.16
-				else
-					ctype = Gdk.CursorType.Arrow;
-				
+				var cursors = new Dictionary<CursorType, Gdk.CursorType> {
```

Comment:
Why should we use an expensive `Dictionary` here? `if/else` or a switch is much more efficient.